packages/email-components/src/components/RatingItem/RatingItem.scss

```diff
@@ -0,0 +1,48 @@
+@use 'theme';
+@use '../../../node_modules/@lmc-eu/spirit-web/scss/tools/dictionaries' as dictionaries;
```

Comment:
Is the `dictionaries` used anywhere here?

Reply:
no